recipe/meta.yaml

```diff
@@ -1,113 +1,153 @@
-      #- issue-7567.patch
-  # these are git submodules from the {{ version }} release
-  # https://github.com/protocolbuffers/protobuf/tree/v{{ version }}/third_party
-  - url: https://github.com/google/benchmark/archive/5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8.tar.gz
-    sha256: 5dc92703f811f94e2aa63bdab07ab749f28a094befa6cdfd5fe177f947590a48
-    folder: third_party/benchmark
-  - url: https://github.com/google/googletest/archive/5ec7f0c4a113e2f18ac2c6cc7df51ad6afc24081.tar.gz
-    sha256: 0e2f36e8e403c125fd0ab02171bdb786d3b6b3875b6ccf3b2eb7969be8faecd0
-    folder: third_party/googletest
+  - url: https://github.com/protocolbuffers/protobuf/archive/refs/tags/v{{ version.replace(".rc", "-rc") }}.tar.gz
+    sha256: a700a49470d301f1190a487a923b5095bf60f08f4ae4cac9f5f7c36883d17971
+    patches:
+      - patches/0001-use-consistent-cmake-location.patch
+      - patches/0002-set-static-lib-extension-on-windows.patch
+      # backport https://github.com/protocolbuffers/protobuf/pull/12577
+      - patches/0003-Enable-the-selection-of-system-provided-jsoncpp-1257.patch
+      - patches/0004-always-look-for-shared-abseil-builds.patch
+      # grpc_cpp_java_plugin links against otherwise internal symbols
+      - patches/0005-Export-functions-in-google-compiler-java-names.patch
+      - patches/0006-do-not-install-vendored-gmock.patch
 
 build:
   number: 0
 
-requirements:
-  build:  # [linux or (osx and arm64)]
-    - patch  # [linux]
-    - sed  # [osx and arm64]
-
 outputs:
   - name: libprotobuf
-    script: build-shared.sh  # [unix]
-    script: bld-shared.bat  # [win]
+    script: build-lib.sh  # [unix]
+    script: build-lib.bat  # [win]
     build:
       missing_dso_whitelist:  # [s390x]
         - $RPATH/ld64.so.1    # [s390x]
       run_exports:
-        # breaks backwards compatibility and new SONAME each minor release
-        # https://abi-laboratory.pro/tracker/timeline/protobuf/
-        # One exception was that 3.6.1 was incompatible with 3.6.0
-        - {{ pin_subpackage('libprotobuf', max_pin='x.x') }}
+        # protobuf now (intentionally) increments the SOVER with every patch release, which
+        # breaks tools like grpc_plugin_cpp if they get the wrong libprotoc at runtime, see
+        # https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/4075
+        - {{ pin_subpackage('libprotobuf', max_pin='x.x.x') }}
+      ignore_run_exports_from:
+        - jsoncpp
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - cmake  # [win]
-        - ninja  # [win]
-        - autoconf  # [not win]
-        - automake  # [not win]
-        - libtool  # [not win]
-        - pkg-config  # [not win]
-        - unzip  # [not win]
-        - make  # [not win]
-        - sed  # [osx and arm64]
+        - cmake
+        - ninja
+        - patch                # [unix]
+        - m2-patch             # [win]
       host:
+        - gtest 1.14.0
```

Comment:
gtest has a run export, which means that libprotobuf will have a runtime requirement on gtest. We probably don't want that.

Reply:
yes, added it to ignore_run_exports